BUILD.md

```diff
@@ -43,3 +43,5 @@ make 2.6.0-alpine
 
 It is also possible to just build the image without testing with `make build_2.6.0`, just testing: `make test_2.6.0`, running the image: `make run_2.6.0` or even run the image for a shell `make runsh_2.6.0`.
 
+You can also override the artemis binary distribution url by passing a parameter containing a zip file containing the distribution.  ie.  `make BUILD_ARGS="--build-arg ACTIVEMQ_DISTRO_URL=http://hostname/amq-broker-7.2.0-bin.zip" 2.6.0`
```

Comment:
Please use an ARTEMIS zip example to not to suggest RH support.

Reply:
Also I would comment that `ACTIVEMQ_ARTEMIS_VERSION` has to be set regardless of setting the `ACTIVEMQ_DISTRO_URL` to enable the entrypoint to make decisions.